lib/commits.js

```diff
@@ -12,7 +12,7 @@ module.exports = tags =>
     const [release, parent] = tags
 
     if (!release || !parent || !parent.hash || !release.hash) {
-      reject(new Error('the first release should be created manually.'))
+      reject(new Error('the first two releases should be created manually.'))
```

Comment:
Oh, i mean that literally `[release, parent]` means two consequent tags. I just wanted to stress on requirement of two tags, because it was not obvious for me why it does not work after i created a tag manually. I can revert it.